### PR TITLE
Riff raff yaml

### PR DESCRIPTION
**Pull request by tomrf1**
Already tested in PROD

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -19,6 +19,8 @@ object FloodgateBuild extends Build {
     scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-target:jvm-1.8")
   )
 
+  def env(key: String): Option[String] = Option(System.getenv(key))
```
Comment (annebyrne) [inline project/Build.scala]: Why have you included this @tomrf1 ?

**Comment by annebyrne** [comment]
@tomrf1 Can we chat about the env variable?

**Comment by annebyrne** [comment]
Tested on `PROD` with further changes

**Comment by tomrf1** [comment]
👍

**Pull request merged**